### PR TITLE
feat(core): integrate basic sentinel

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

integrate the basic sentinel into experience for password and verification code flows

## Testing

image

tested locally (same process as the new integration tests)

## Checklist

- [x] `.changeset`
- [ ] ~~unit tests~~ (integration tests are better to handle this)
- [x] integration tests
- [ ] ~~docs~~

### CODE REVIEW

File: packages/integration-tests/src/ui-helpers/index.ts [hunk 1]
```diff
@@ -146,3 +150,43 @@ export const expectToClickSidebarMenu = async (page: Page, menuText: string) =>
...
+ * Setup the email connector and update the sign-in experience to the following:
+ *
+ * - Sign-in and register mode
+ * - Use username and password to sign-up
+ * - Use username or email to sign-in
+ * - Email sign-in can use verification code
+ *
+ * @param passwordPolicy The password policy to partially update the existing one.
+ */
+export const setupStandardExperience = async (passwordPolicy?: PartialPasswordPolicy) => {
```
Comment (wangsijie) [inline packages/integration-tests/src/ui-helpers/index.ts:164]: nit: not sure on this "standard" experience, could we have a better name? Or can we have agreements on the meaning of "standard".

**Review by wangsijie — APPROVED** [review]

File: packages/integration-tests/src/ui-helpers/index.ts (hunk 1 above)
Comment (gao-sun) [reply packages/integration-tests/src/ui-helpers/index.ts:164]: you are right. let me think about it.

**Review by darcyYe — APPROVED** [review]

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by wangsijie — APPROVED** [review]

File: packages/integration-tests/src/ui-helpers/index.ts (hunk 1 above)
Comment (gao-sun) [reply packages/integration-tests/src/ui-helpers/index.ts]: updated to `setupUsernameAndEmailExperience` to be more specific

**Pull request merged**